app/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.tsx

```diff
@@ -59,7 +57,7 @@ const TableBranches = ({
       ) : (
         <>
           {savedBranches.map((branch, key) => (
-            <TableBranch key={key} {...{ hasSchema, experiment, branch }} />
```

Comment:
We no longer need to know if it has a schema?

Reply:
That's actually a holdover from a long since passed intermediate implementation where we thought we only wanted to enter feature values for features that were enabled and had schemas, but that concept has gone and away and this should've been removed a while ago.